src/app/components/home/details/details.component.ts

```diff
@@ -59,8 +70,8 @@ export class DetailsComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.firstFilePath = encodeURI('file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/thumbnails/' + this.imgId + '.jpg');
-    this.fullFilePath =  encodeURI('file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/filmstrips/' + this.imgId + '.jpg');
+    this.firstFilePath = 'file://' + this.folderPath + '/' + 'vha-' + this.hubName + '/thumbnails/' + this.imgId + '.jpg';
```

Comment:
I removed `encodeURI` because it breaks the ability to view images on Windows. 

I'll figure out a way to both allow folders with spaces in their names _and_ continued Windows support in a future PR 👌